Tests/Integration/Components/CategoryResolver/AutoCategoryResolverTest.php

```diff
@@ -61,7 +61,7 @@ public function testResolve()
         $this->createRemoteCategories($createdCategories);
 
         if (!$bootsCategory) {
-            $this->categoryResolver->createLocalCategory('Boots', '/deutsch/boots', $defaultCategory->getId(), 1234);
+            $this->categoryResolver->createLocalCategory('Boots', '/deutsch/boots', $defaultCategory->getId(), 1234, 'test');
```

Comment:
I think more speaking name for stream is better, because right now we know it, but after some months string 'test' means every think. 'Boots stream' for example is better.